reaktive/src/nativeCommonMain/kotlin/com/badoo/reaktive/scheduler/CreateTrampolineScheduler.kt

```diff
@@ -0,0 +1,3 @@
+package com.badoo.reaktive.scheduler
+
+actual fun createTrampolineScheduler(): Scheduler = TODO("No trampoline scheduler so far :-(")
```

Comment:
Can we resuse `MainScheduler` with warning output in console? As I remember, we decided to not crash app, if something can't be implemented on platform. Otherwise client code will be covered with a lot of checks `if (platform) oneScheduler else otherScheduler`.

Reply:
I believe we should just implement it :-) Main scheduler violates Trampoline scheduler rules (it should execute in the same callstack)

Reply:
But yes, better to use MainScheduler

Reply:
Not sure how to produce a warning, let's keep it like this and then just implement it? It's quite easy.

Reply:
https://github.com/badoo/Reaktive/issues/41 Created issue, so we won't forget about it before release. I think it is good idea to create issues for each TODO and goals to make roadmap and process visible.